src/main/java/com/team4/testingsystem/dto/QuestionDTO.java

```diff
@@ -8,6 +12,17 @@
     private String creator;
     private String level;
     private String module;
+    private List<String> answers;
+
+    public QuestionDTO(Question question) {
+        this.questionBody = question.getBody();
+        this.isAvailable = question.isAvailable();
+        this.creator = question.getCreator().getName();
+        this.level = question.getLevel().getName();
+        this.module = question.getModule().getName();
+        this.answers = new ArrayList<>();
+        question.getAnswers().forEach(answer -> answers.add(answer.getAnswerBody()));
```

Comment:
question.getAnswers().map(answer::getAnswerBody).collect(Collectors.toList());

Reply:
BTW, we will need an option to get questions with correct mark (for coaches edit)